### PR TITLE
Moa Hammarbäck - Happy API 

### PR DESCRIPTION
**Pull request by mhammarback**
I think making the endpoints is ok and there is good material about the mongoose model but i feel that the most challenging part is to set up the database and i'm not comfortable with it yet.   

my project backend : https://nice-thoughts-api.herokuapp.com/thoughts
my project connected to frontend: https://happy-booth-d6f66f.netlify.app/

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,13 +1,32 @@
...
+
+## TECH 
+- MongoDB
+- Mongoose
+- Node.js
+- Express
 
 ## View it live
+https://nice-thoughts-api.herokuapp.com/thoughts
+
```
Comment (karanmann) [inline README.md:32]: Very detailed Readme Moa, Great Job 👍

File: server.js [hunk 2]
```diff
@@ -7,23 +7,61 @@ const mongoUrl = process.env.MONGO_URL || "mongodb://localhost/happyThoughts"
...
-// Defines the port the app will run on. Defaults to 8080, but can be 
-// overridden when starting the server. For example:
-//
-//   PORT=9000 npm start
-const port = process.env.PORT || 8080
+const Thought = mongoose.model('Thought', {
+  message: {
+    type: String,
+    required: true,
+    minlength: 5,
```
Comment (karanmann) [inline server.js:14]: Validations implemented 👍

File: server.js [hunk 3]
```diff
@@ -7,23 +7,61 @@ const mongoUrl = process.env.MONGO_URL || "mongodb://localhost/happyThoughts"
...
+    res.status(201).json(savedThought)
+  } catch (err) {
+  res.status(400).json({ message: 'could not post happy thought', errors:err.errors})
+ }
+})
+
+app.post('/thoughts/:thoughtId/like', async (req, res) => { 
+  const thoughtId = req.params.thoughtId
+
+  try {
```
Comment (karanmann) [inline server.js:57]: Great Job Moa and congratulations on completing the blue level. The functionality work both on the front and the back end well. The code was clean and easy to read. 🥇 💯

**Review by karanmann — APPROVED** [review]

**Review by KarolCislowski — APPROVED** [review]
Great work 👍🏻

**Pull request closed without merging**